ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/CJavaDriverBasePlugin.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2017 Netflix, Inc.  All rights reserved.
```

Comment:
I am not sure this is the right copyright note we use. You can use `addLicense` from the Gradle wrapper to do that automatically.